pyerrors/input/json.py

```diff
@@ -395,7 +400,17 @@ def get_Array_from_dict(o):
         return np.reshape(ret, layout)
 
     def get_Corr_from_dict(o):
-        taglist = o.get('tag')
+        if isinstance(o.get('tag'), list):  # supports the old way
+            taglist = o.get('tag')  # This had to be modified to get the taglist from the dictionary
+            temp_prange = None
+        else:
```

Comment:
For safety reasons I would suggest to check whether `o.get('tag')` is a `dict` and raise an Exception if the tag is neither `list` nor `dict`.

Reply:
Yes, i added the exception

Reply:
I agree. Too bad, that we already have to care about backwards compatibility, but I hope, we don't have to include too many of such cases. The tag-dictionary will certainly help in this regard.

Reply:
I guess as long as it is pre 2.0, Fabian could decide to break the format. 
One could easily update data into the new format.